src/jsdoc_transformer.ts

```diff
@@ -162,28 +157,39 @@ export function maybeAddHeritageClauses(
           tagName = 'extends';
         } else {
           // Closure can only @implements an interface, not a class.
-          mtt.debugWarn(decl, `omitting @implements of a class: ${expr.getText()}`);
-          return null;
+          warn(decl, `dropped extends of class: ${expr.getText()}`);
```

Comment:
this is an `implements` here from the perspective of the TS user, right?

Reply:
Good catch, fixed.

Reply:
Good catch, fixed.